Package.swift

```diff
@@ -0,0 +1,25 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "NMSSH",
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "NMSSH",
+            targets: ["NMSSH"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+    ],
+	targets: [
+ 		.binaryTarget(
+ 			name: "NMSSH",
+ 			url: "https://github.com/DwayneCoussement/NMSSH/releases/download/2.3.3/NMSSH.xcframework.zip",
```

Comment:
This should probably be pulled from NMSSH's github, no?

Reply:
For sure, building this framework should be part of a release in that case.